tests/testthat/test-R6_inherited_methods.R

```diff
@@ -0,0 +1,20 @@
+test_that("urls to inherited methods of R6 classes are correctly modified ", {
+  skip_if_no_pandoc()
```

Comment:
I think it would be better to start with a direct test of `fix_R6_inherited_hrefs()`, and then figure out if it's necessary to install and document a complete package.

Reply:
I made the tests a lot more minimal.